src/manuel/tests/doctests/myst/myst-markdown.md

```diff
@@ -0,0 +1,62 @@
+# MyST markdown
```

Comment:
I think—at least for the time being—it would be best to write this test/document in ReST so that you can use the full set of Manuel mechanisms.  In particular, the [capture](https://pythonhosted.org/manuel/#capturing-blocks) tool so that you can write MyST inside the document, capture it and then run code/tests on it.

Reply:
The MyST documentation file is now included in tests. See  `src/manuel/tests.py`.

Reply:
I am in a clinch now. I tried to make "Capturing Blocks" fit for MyST, but failed in an acceptable amount of time.  But rewriting the short  documentation in rst does not feel well for me. 
What's your opinion? I follow your opinion as maintainer.